projects/cashmere/src/lib/sass/button.scss

```diff
@@ -67,6 +63,30 @@ $btn-icon-sz: 18px;
     }
 }
 
+@mixin hc-button-sm() {
+    padding: 3px 12px;
+    height: 26px;
+    min-width: auto;
+    @include fontSize(14px);
+    line-height: 14px;
+}
+
+@mixin hc-button-md() {
+    padding: 0 20px;
+    height: 35px;
+    min-width: 150px;
+    @include fontSize(15px);
+    line-height: 15px;
+}
+
+@mixin hc-button-lg() {
+    padding: 0 30px;
+    height: 40px;
+    min-width: 200px;
+    @include fontSize(18px);
```

Comment:
Sorry for being nitpicky - I just noticed this now with the 40px size, 18 font size looks a bit big to me.  Can we change this to 17px instead?  Then I think we're good.  Thanks!